Gemfile

```diff
@@ -43,6 +43,8 @@ end
 group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
+  gem 'wraith', '~> 3.0'
+  gem 'phantomjs', '~> 2.0'
```

Comment:
Not sure we need the phantomjs dependency here as it's preloaded on the VM.

:+1: For combining the Gemfiles.